.travis.yml

```diff
@@ -40,6 +40,21 @@ script:
 - bundle exec rake puppetserver
 after_script:
 - docker-compose -f spec/docker-compose.yml logs
+deploy:
```

Comment:
Does this need to be nested under `jobs:` as a multi stage build?

Reply:
My understanding is that this is a "phase" not a "stage"
https://docs.travis-ci.com/user/job-lifecycle/#the-job-lifecycle